docs/lang-ref-structs.md

```diff
@@ -0,0 +1,103 @@
+### Structs
+
+Structs are well-defined data objects, comprised of key/value pairs known as "attributes". They are a way to store and refer to data of a known structure.
```

Comment:
should add version availability for this feature.

Reply:
Do you mean retroactively? I _did_ note in places where the index notation is referenced that this is available "as of v0.31.0"

Reply:
nm, didnt see index notation has a version notice at the end of the line.